applications/ecallmgr/src/ecallmgr_fs_authn.erl

```diff
@@ -192,7 +192,9 @@ handle_directory_lookup(Id, Props, Node) ->
 -spec lookup_user(atom(), ne_binary(), ne_binary(), kz_proplist()) -> fs_handlecall_ret().
 lookup_user(Node, Id, Method,  Props) ->
     Domain = props:get_value(<<"domain">>, Props),
-    [Realm|_] = binary:split(get_auth_realm(Props), [<<":">>, <<";">>]),
+    AuthRealm = get_auth_realm(Props),
+    lager:debug("auth realm: '~p'", [AuthRealm]),
+    [Realm|_] = binary:split(AuthRealm, [<<":">>, <<";">>]),
```

Comment:
lets protect the call here
case binary:split...
else -> ecallmgr_fs_xml:not_found()